tests/unit/Statement/StatementListTest.php

```diff
@@ -42,7 +42,7 @@ private function getStatement( $propertyId, $guid, $rank = Statement::RANK_NORMA
 
 		$statement->expects( $this->any() )
 			->method( 'getPropertyId' )
-			->will( $this->returnValue( PropertyId::newFromNumber( $propertyId ) ) );
+			->will( $this->returnValue( new PropertyId( 'P' . $propertyId ) ) );
```

Comment:
In production code I would agree, but this is a test. All numbers used here are integer constants from test providers in this file, and never user input.